lxd/project/limits/permissions.go

```diff
@@ -1677,24 +1677,24 @@ func AllowClusterUpdate(ctx context.Context, tx *db.ClusterTx, sysinfo map[strin
 		return nil
 	}
 
-	aggregates, err := getClusterMemberAggregateLimits(ctx, tx, newConfig, requiredLimits)
+	clusterMemberNames := []string{}
```

Comment:
When you're appending to a slice there's no need to initialise it first, so can just do `var clusterMemberNames []string`